packages/react-components/src/components/VideoTile.tsx

```diff
@@ -129,18 +139,40 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
           )}
         </Stack>
       )}
-      {displayName && showDisplayName && (
-        <Text
-          className={mergeStyles(
-            isVideoReady ? videoHint : disabledVideoHint,
-            // when video is on, the displayName has a grey-ish background, so no use of theme
-            { color: isVideoReady ? palette.neutralPrimary : theme.palette.neutralPrimary },
-            styles?.displayNameContainer
+
+      <Stack
+        horizontal
+        className={mergeStyles(
+          isVideoReady ? videoHint : disabledVideoHint,
+          // when video is on, the displayName has a grey-ish background, so no use of theme
+          { color: isVideoReady ? palette.neutralPrimary : theme.palette.neutralPrimary },
+          styles?.displayNameContainer
+        )}
+      >
+        <Stack.Item>
+          {displayName && showDisplayName && (
+            <Text
+              style={{
+                fontSize: '12px',
+                fontWeight: 600,
+                color: isVideoReady ? palette.neutralPrimary : theme.palette.neutralPrimary
+              }}
+            >
+              {displayName}
+            </Text>
           )}
-        >
-          {displayName}
-        </Text>
-      )}
+        </Stack.Item>
+        <Stack.Item style={{ paddingLeft: '4px' }}>
```

Comment:
do we want to move this style into a style file and also make sure we convert px to rem for font scaling

Reply:
Careful with anything that does paddingLeft or paddingRight - will this work in RTL?

Reply:
Thats a great point to bring up James!

https://rtlstyling.com/posts/rtl-styling/

.media[dir="rtl] img {
    float: right;
    margin-right: 0;
    margin-left: 16px;
}

We may need to address rtl specific areas where we are explicitly doing things on the left or the right.

Reply:
Fixed it through a more sensible styling.